packages/commonwealth/client/scripts/views/modals/order_topics_modal/order_topics_modal.tsx

```diff
@@ -10,14 +10,17 @@ import { CWIconButton } from 'views/components/component_kit/cw_icon_button';
 
 import DraggableTopicsList from './draggable_topics_list';
 import { CWText } from '../../components/component_kit/cw_text';
+import {
+  useFetchTopicsQuery,
+  useUpdateFeaturedTopicsOrderMutation,
+} from 'state/api/topics';
 
 type OrderTopicsModalProps = {
   onModalClose: () => void;
 };
 
-const getSortedTopics = (): Topic[] => {
-  const topics = app.topics.store
-    .getByCommunity(app.chain.id)
+const getSortedTopics = (rawTopics: Topic[]): Topic[] => {
```

Comment:
it existed before but the name seems a little misleading, can we change it to `getFilteredTopics`?

Reply:
agree